src/documents/firstDiscoveryServer.md

```diff
@@ -0,0 +1,365 @@
+---
+title: First Discovery Server
+layout: default
+category: Overview
+---
+
```

Comment:
It might be helpful to have a brief introductory paragraph giving a quick explanation of what the FD Server is, what is its relationship to Kettle, etc.
It also might be helpful to reorder the high-level content, and start with configuration, then launching. That would probably flow a bit better for the reader.